### PR TITLE
9564 feedback on formalized relations

### PR DESCRIPTION
**Pull request by travislondon**

### CODE REVIEW

File: doc-bridgepoint/notes/9564_feedback_on_formalized_relations/9564_feedback_on_formalized_relations.int.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+# Feedback for formalized associations  
+### xtUML Project Implementation Note
+
+### 1. Abstract
+
+This note describes the files changed for this work.
+
+### 2. Document References
+
+In this section, list all the documents that the reader may need to refer to.
```
Comment (keithbrown) [inline doc-bridgepoint/notes/9564_feedback_on_formalized_relations/9564_feedback_on_formalized_relations.int.md]: Remove line

File: doc-bridgepoint/notes/9564_feedback_on_formalized_relations/9564_feedback_on_formalized_relations.int.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+None.  
+
+### 7. Unit Test
+
+See [[2.2]](#2.2).  
+
+### 8. User Documentation
+
+Describe the end user documentation that was added for this change.
```
Comment (keithbrown) [inline doc-bridgepoint/notes/9564_feedback_on_formalized_relations/9564_feedback_on_formalized_relations.int.md]: Add text you sent to me in chat on July 3.  Note that it should be added to git/bridgepoint/src/org.xtuml.bp.doc/Reference/UserInterface/xtUMLModeling/ModelElements/HTML/ClassDiagram.htm

Then, let's work together to get that file updated and included in this pull request.

File: doc-bridgepoint/notes/9564_feedback_on_formalized_relations/9564_feedback_on_formalized_relations.int.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+See [[2.2]](#2.2).  
+
+### 6. Implementation Comments
+
+None.  
+
+### 7. Unit Test
+
+See [[2.2]](#2.2).
```
Comment (keithbrown) [inline doc-bridgepoint/notes/9564_feedback_on_formalized_relations/9564_feedback_on_formalized_relations.int.md]: Add a description of/pointer to the model created for this test.  Was a JUnit added?  If so, call it out.  Or, is it a manual test?

File: doc-bridgepoint/notes/9564_feedback_on_formalized_relations/9564_feedback_on_formalized_relations.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+To easily distinguish between informal and formal associations we shall take the approach of drawing formal associations as bold.
+
+6.1.1 Introduce Style::Bold  
+6.1.2 When drawing a connector if the line style is Bold, set the line width to 2  
+6.1.3 In Association.get_connector_style() return Style::Bold if isFormal() is true   
+6.1.4 Adjust Gr_c in ui.canvas to support the Bold style for compartment drawing  
+6.1.5 Change ModelClass.get_compartment_style to return Bold for entries at the right if the attribute is referential  
+
+### 7. Design Comments
+
```
Comment (keithbrown) [inline doc-bridgepoint/notes/9564_feedback_on_formalized_relations/9564_feedback_on_formalized_relations.md]: None.

File: doc-bridgepoint/notes/9564_feedback_on_formalized_relations/9564_feedback_on_formalized_relations.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+### 8. User Documentation
+
```
Comment (keithbrown) [inline doc-bridgepoint/notes/9564_feedback_on_formalized_relations/9564_feedback_on_formalized_relations.md]: Note that the documentation will be updated with a description of the feature and preference.

**Comment by travislondon** [comment]
Changes have been made, other than to the documentation files.

**Comment by keithbrown** [comment]
Due to version mismatches, this needs to have xtuml/master merged into the branch in order to run builds that use the tests.

**Pull request merged**